contracts/RiskManagerV1.sol

```diff
@@ -68,11 +82,13 @@ contract RiskManagerV1 is Auctioneer, Ownable {
 
     constructor(
         IERC20 _tbtcToken,
+        ISignerBondsSwapStrategy _signerBondsSwapStrategy,
```

Comment:
Nitpick but I'd keep this parameter after `_coveragePool`. This is a configuration-related parameter, same as the last two ones.